packages/trees/frontend/sirius-components-trees/src/trees/Tree.tsx

```diff
@@ -10,11 +10,20 @@
  * Contributors:
  *     Obeo - initial API and implementation
  *******************************************************************************/
+import { makeStyles } from '@material-ui/core/styles';
 import { useEffect, useRef } from 'react';
 import { TreeItem } from '../treeitems/TreeItem';
 import { TreeProps } from './Tree.types';
 
+const useTreeStyle = makeStyles((_) => ({
+  ul: {
+    width: 'fit-content',
+    minWidth: '-webkit-fill-available; min-width: -moz-available',
```

Comment:
I don't feel confortable moving forward with CSS values that are barely out of a working draft for some and others which are still in a working draft. If a child DOM element needs to occupy a specific space, then its parent should determine how to layout it